### DISCUSSION TITLE
Change URL search bar for Dropdown Version

### DISCUSSION THREAD

**Discussion by GasparB123** [DarkaOnLine/L5-Swagger#612] [Q&A]
The Swagger UI for L5 Swagger in Laravel by default shows a URL search bar:

![Captura de pantalla 2024-07-16 101301](https://github.com/user-attachments/assets/214d73c5-bc56-45b1-99ac-d549db0b7df9)

But I want it to show a select definition dropdown that let's me select the version (v1,v2,v3,etc.) (I took this picture from a C# API):

![Captura de pantalla 2024-07-16 101246](https://github.com/user-attachments/assets/4edebfd1-0b31-4c3f-b4b9-b73a495fa6bf)

How do I code or change the configuration for this?? I've done Laravel for a while now, but just recently started analising using Swagger

**Comment by jpedryc** [comment]
@GasparB123 I'm looking for the exact thing. How did you actually manage to have it as a search bar? :thinking: Do you have an example config?

I don't have any black header at the top - is it not the BaseLayout?
<img width="758" alt="image" src="https://github.com/user-attachments/assets/d389267c-a418-47f3-93bf-2383ab3ab897">
